dev/src/index.ts

```diff
@@ -1290,15 +1330,7 @@ function validateDocumentData(obj, options) {
  * @param {string} Log message
  */
 
-/**
- * Sets the log function for all active Firestore instances.
- *
- * @method Firestore.setLogFunction
- * @param {Firestore~logFunction} logger - A log function that takes a single
- * string.
- */
-Firestore.setLogFunction = setLogFunction;
-
+// tslint:disable-next-line:no-default-export
```

Comment:
It looks like you ended up adding this below...

Reply:
For some reason, the JSDoc comment itself triggers this warning, as well as the actual line that defines it.  I suspect it has to do with

```
@module {Firestore} @google-cloud/firestore
```